### PR TITLE
source the rc.

### PR DESCRIPTION
**Pull request by hemanth**
**Summary**
Instead of requesting the user to open a new terminal.

**Test plan**
N.A

### CODE REVIEW

**Comment by hemanth** [comment]
`travis` is failing due to timeouts, not related to this PR.

File: scripts/install-latest.sh [hunk 1]
```diff
@@ -41,8 +41,8 @@ yarn_link() {
     printf "$cyan> We've added the following to your $YARN_PROFILE\n"
     echo "> If this isn't the profile of your current shell then please add the following to your correct profile:"
     printf "   $SOURCE_STR$reset\n"
-
-    printf "$green> Successfully installed! Please open another terminal where the \`yarn\` command will now be available.$reset\n"
+    source $YARN_PROFILE
+    printf "$green> Successfully installed! `yarn` command will now be available. $reset\n"
```
Comment (mathiasbynens) [inline scripts/install-latest.sh]: Completely unrelated to this patch, but “will now” should really be “is”

Comment (hemanth) [inline scripts/install-latest.sh]: Successfully installed!  You may `yarn` now. 

^ How does it sound ;) ?

**Comment by kittens** [comment]
Profiles aren't necessarily safe to execute multiple times in the same shell session so I'm not sure this is a good idea.

**Comment by hemanth** [comment]
> Profiles aren't necessarily safe to execute multiple times in the same shell session

Agree but in this scenario it seems safe?

File: scripts/install-latest.sh (hunk 1 above)
Comment (ZakariaRidouh) [inline scripts/install-latest.sh]: @hemanth I think this would be slightly better:

Successfully installed! You can run `yarn` now.

Comment (hemanth) [inline scripts/install-latest.sh]: Let us see, if this PR is going in at all!

**Comment by kittens** [comment]
Why do you think in this scenario it seems safe? The profile code will be executed twice, all variables will be exported twice. ie. if you have `export PATH="$PATH:/whatever"` in your profile then it'll be executed twice duplicating the variable as well as many other issues. It's fundamentally unsafe.

**Comment by mathiasbynens** [comment]
Honest question: what’s “unsafe” about repeating `export PATH="$PATH:/whatever"`? Trying to imagine an attack scenario here and failing.

**Comment by bestander** [comment]
We could give user an option to run `source $YARN_PROFILE` or open terminal.

**Comment by bestander** [comment]
I talked to team mates, a few people were not happy about executing some scripts from .bashrc multiple times so let's find another way.
I think a recommendation to open a new terminal or running `source .bashrc` manually could be a reasonable alternative

**Comment by Daniel15** [comment]
Running `exec $SHELL -l` should be a safer way to accomplish the same thing - It'll start a new shell process and the `-l` flag runs it as a login shell so it'll execute the profile script.

**Pull request closed without merging**